spot_hardware_interface/include/spot_hardware_interface/spot_constants.hpp

```diff
@@ -29,19 +29,19 @@ inline constexpr int kNjointsArm = 19;
 // Gain values https://github.com/boston-dynamics/spot-cpp-sdk/blob/master/cpp/examples/joint_control/constants.hpp
 // This will be handled via a parameter in the future so there is the option to change them, for now they are hardcoded
 
-/// @brief Default Kp gains for robot without an arm
-inline constexpr float kDefaultKpNoArm[] = {624.0, 936.0, 286.0, 624.0, 936.0, 286.0,
-                                            624.0, 936.0, 286.0, 624.0, 936.0, 286.0};
+/// @brief Default k_q_p gains for robot without an arm
+inline constexpr float kDefaultKqpNoArm[] = {624.0, 936.0, 286.0, 624.0, 936.0, 286.0,
+                                             624.0, 936.0, 286.0, 624.0, 936.0, 286.0};
 
-/// @brief Default Kd gains for robot without an arm
-inline constexpr float kDefaultKdNoArm[] = {5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04};
+/// @brief Default k_qd_p gains for robot without an arm
+inline constexpr float kDefaultKqdpNoArm[] = {5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04};
 
-/// @brief Default Kp gains for robot with an arm
-inline constexpr float kDefaultKpArm[] = {624.0, 936.0, 286.0,  624.0, 936.0, 286.0, 624.0, 936.0, 286.0, 624.0,
-                                          936.0, 286.0, 1020.0, 255.0, 204.0, 102.0, 102.0, 102.0, 16.0};
+/// @brief Default k_q_p gains for robot with an arm
+inline constexpr float kDefaultKqpArm[] = {624.0, 936.0, 286.0,  624.0, 936.0, 286.0, 624.0, 936.0, 286.0, 624.0,
```

Comment:
It seems that currently the default gains for the leg joints are the same for both robots. Should we just keep one vector of the leg joint gains in that case if we are not going to change these default gains?

Reply:
Yeah, I thought about this too. I think structuring it in this way makes it easier to handle the logic in the hardware interface. so instead of parsing together a list of the gains from two separate sources (i.e. a list of arm gains and a list of leg gains), I can just get all of the gains from the same variable at once.

Reply:
I see. Maybe we could at least mention in the comment that the numerical values are identical currently.